internal/istiooperator/istio-operator-light.yaml

```diff
@@ -253,7 +253,6 @@ spec:
         enabled: false
       network: ""
       omitSidecarInjectorConfigMap: false
-      oneNamespace: false
```

Comment:
If this field is configured install will fail. It was removed from Istio, because it wasn't used for a long time. Only found information about this in Istio repository.

Reply:
okey, thanks